utils/python/CIME/SystemTests/pem.py

```diff
@@ -31,7 +31,7 @@ def build(self, sharedlib_only=False, model_only=False):
             logging.warn("Starting bld %s"%bld)
             self._case.flush()
             case_setup(self._caseroot,reset=True)
-            run_cmd('case.build --clean')
+            run_cmd('./case.build --clean')
```

Comment:
Shouldn't you add  from_dir=self._caseroot to  be consistent with run_cmd calls in pea.py , ncr.py , ....